更新galaxyspace后服务器无法加载。

```
[12:51:57] [Server thread/ERROR] [FML]: Fatal errors were detected during the transition from AVAILABLE to SERVER_ABOUT_TO_START. Loading cannot continue
[12:51:57] [Server thread/ERROR] [FML]:
        States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
        UCHIJAA mcp{9.05} [Minecraft Coder Pack] (minecraft.jar)
        UCHIJAA FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1614-1.7.10-universal.jar)
        UCHIJAA Forge{10.13.4.1614} [Minecraft Forge] (forge-1.7.10-10.13.4.1614-1.7.10-universal.jar)
        UCHIJAA appliedenergistics2-core{rv3-beta-503-GTNH} [Applied Energistics 2 Core] (minecraft.jar)
        UCHIJAA CodeChickenCore{1.3.10} [CodeChicken Core] (minecraft.jar)
        UCHIJAA CropLoadCoreASM{0.0.2} [CroploadCore ASM Core] (minecraft.jar)
        UCHIJAA NotEnoughItems{2.7.0-GTNH} [NotEnoughItems] (NotEnoughItems-2.7.0-GTNH.jar)
        UCHIJAA OmniOcularUnofficialCore{GRADLETOKEN_VERSION} [Omni Ocular Unofficial Core] (minecraft.jar)
        UCHIJAA OpenComputers|Core{1.11.1-GTNH} [OpenComputers (Core)] (minecraft.jar)
        UCHIJAA MobiusCore{1.4.5-mapless} [MobiusCore] (minecraft.jar)
        UCHIJAA PlayerAPI{1.5.0} [Player API] (minecraft.jar)
        UCHIJAA ThE-core{1.0.0.1} [Thaumic Energistics Core] (minecraft.jar)
        UCHIJAA GTNHLib Core{0.5.23} [GTNHLib Core] (minecraft.jar)
        UCHIJAA ThaumicTinkerer-preloader{0.1} [Thaumic Tinkerer Core] (minecraft.jar)
        UCHIJAA WitcheryExtras_ASM{0.1-Beta} [WitcheryExtras_ASM] (minecraft.jar)
        UCHIJAA OpenModsCore{0.10.9} [OpenModsCore] (minecraft.jar)
        UCHIJAA <CoFH ASM>{000} [CoFH ASM] (minecraft.jar)
        UCHIJAA unimixins{0.1.19} [UniMixins] (+unimixins-all-1.7.10-0.1.19.jar)
        UCHIJAA unimixins-mixin{0.1.19} [UniMixins: Mixin (UniMix)] (+unimixins-all-1.7.10-0.1.19.jar)
        UCHIJAA unimixins-compat{0.1.19} [UniMixins: Compatibility] (+unimixins-all-1.7.10-0.1.19.jar)
        UCHIJAA mixingasm{0.3} [UniMixins: Mixingasm] (+unimixins-all-1.7.10-0.1.19.jar)
        UCHIJAA spongemixins{2.0.1} [UniMixins: SpongeMixins] (+unimixins-all-1.7.10-0.1.19.jar)
        UCHIJAA mixinbooterlegacy{1.2.1} [UniMixins: MixinBooterLegacy] (+unimixins-all-1.7.10-0.1.19.jar)
        UCHIJAA gasstation{0.5.1} [UniMixins: GasStation] (+unimixins-all-1.7.10-0.1.19.jar)
        UCHIJAA gtnhmixins{2.2.0} [UniMixins: GTNHMixins] (+unimixins-all-1.7.10-0.1.19.jar)
        UCHIJAA mixinextras{0.1.19} [UniMixins: MixinExtras] (+unimixins-all-1.7.10-0.1.19.jar)
        UCHIJAA IC2{2.2.828-experimental} [IndustrialCraft 2] (industrialcraft-2-2.2.828a-experimental.jar)
        UCHIJAA AdvancedSolarPanel{1.7.10-3.5.1} [Advanced Solar Panels] (AdvancedSolarPanel-1.7.10-3.5.1a.jar)
        UCHIJAA adventurebackpack{1.2.4-GTNH} [Adventure Backpack] (adventurebackpack-1.2.4-GTNH.jar)
        UCHIJAA gtnhlib{0.5.23} [GTNH Lib] (gtnhlib-0.5.23.jar)
        UCHIJAA appliedenergistics2{rv3-beta-503-GTNH} [Applied Energistics 2] (appliedenergistics2-rv3-beta-503-GTNH.jar)
        UCHIJAA Baubles{2.0.3} [Baubles] (BaublesExpanded-2.0.3.jar)
        UCHIJAA CoFHCore{1.7.10R3.1.4} [CoFH Core] (CoFHCore-[1.7.10]3.1.4-329.jar)
        UCHIJAA Thaumcraft{4.2.3.5} [Thaumcraft] (Thaumcraft-1.7.10-4.2.3.5a.jar)
        UCHIJAA Waila{1.8.2} [Waila] (Waila-1.8.2.jar)
        UCHIJAA thaumicenergistics{1.6.27-GTNH} [Thaumic Energistics] (thaumicenergistics-1.6.27-GTNH.jar)
        UCHIJAA ae2wct{1.11.7} [AE2 Wireless Crafting Terminal] (WirelessCraftingTerminal-1.11.7.jar)
        UCHIJAA ae2fc{1.4.0-gtnh} [AE2 Fluid Crafting] (ae2fc-1.4.0-gtnh.jar)
        UCHIJAA bdlib{1.10.0-GTNH} [BD lib] (bdlib-1.10.0-GTNH.jar)
        UCHIJAA ae2stuff{0.9.0-GTNH} [AE2 Stuff] (ae2stuff-0.9.0-GTNH.jar)
        UCHIJAA ae2thing{v1.66} [AE2thing] (ae2thing-v1.66.jar)
        UCHIJAA AFSU{1.3.1-GTNH} [AFSU Mod] (AFSU-1.3.1-GTNH.jar)
        UCHIJAA akashictome{1.1.9} [Akashic Tome] (akashictome-1.1.9.jar)
        UCHIJAA alchgrate{1.2.1-GTNH} [Alchemical Grate] (alchgrate-1.2.1-GTNH.jar)
        UCHIJAA amazingtrophies{1.2.2} [Amazing Trophies] (amazingtrophies-1.2.2.jar)
        UCHIJAA BuildCraft|Core{7.1.39} [BuildCraft] (buildcraft-7.1.39.jar)
        UCHIJAA BuildCraft|Energy{7.1.39} [BC Energy] (buildcraft-7.1.39.jar)
        UCHIJAA Mantle{0.4.2} [Mantle] (Mantle-0.4.2.jar)
        UCHIJAA battlegear2{1.4.1} [Mine & Blade Battlegear 2] (battlegear2-1.4.1.jar)
        UCHIJAA ForgeMultipart{1.5.0} [Forge Multipart] (ForgeMultipart-1.5.0.jar)
        UCHIJAA ExtraUtilities{1.2.12} [Extra Utilities] (extrautilities-1.2.13a.jar)
        UCHIJAA TConstruct{1.12.16-GTNH} [Tinkers' Construct] (TConstruct-1.12.16-GTNH.jar)
        UCHIJAA GalacticraftCore{3.2.8-GTNH} [Galacticraft Core] (Galacticraft-3.2.8-GTNH.jar)
        UCHIJAA GalacticraftMars{3.2.8-GTNH} [Galacticraft Planets] (Galacticraft-3.2.8-GTNH.jar)
        UCHIJAA AWWayofTime{1.6.9} [Blood Magic: Alchemical Wizardry] (BloodMagic-1.6.9.jar)
        UCHIJAA BuildCraft|Factory{7.1.39} [BC Factory] (buildcraft-7.1.39.jar)
        UCHIJAA BuildCraft|Builders{7.1.39} [BC Builders] (buildcraft-7.1.39.jar)
        UCHIJAA Railcraft{9.15.15} [Railcraft] (Railcraft-9.15.15.jar)
        UCHIJAA TwilightForest{2.7.1} [The Twilight Forest] (TwilightForest-2.7.1.jar)
        UCHIJAA chisel{2.16.0-GTNH} [Chisel] (chisel-2.16.0-GTNH.jar)
        UCHIJAA endercore{0.4.6} [EnderCore] (endercore-0.4.6.jar)
        UCHIJAA Natura{2.7.5} [Natura] (Natura-2.7.5.jar)
        UCHIJAA BiomesOPlenty{2.1.0} [Biomes O' Plenty] (BiomesOPlenty-1.7.10-2.1.0.2308-universal.jar)
        UCHIJAA HardcoreEnderExpansion{1.12.0-GTNH} [Hardcore Ender Expansion] (HardcoreEnderExpansion-1.12.0-GTNH.jar)
        UCHIJAA Forestry{4.10.0} [Forestry] (Forestry-4.10.0.jar)
        UCHIJAA EnderIO{2.8.22} [Ender IO] (EnderIO-2.8.22.jar)
        UCHIJAA supersolarpanel{GRADLETOKEN_VERSION} [Super Solar Panel] (Supersolarpanel-1.1.3.jar)
        UCHIJAA YAMCore{0.6.0} [YAMCore] (YAMCore-0.6.0.jar)
        UCHIJAA dreamcraft{2.7.5} [GT: New Horizons] (GTNewHorizonsCoreMod-2.7.5.jar)
        UCHIJAA IronChest{6.0.87} [Iron Chests] (IronChest-6.0.87.jar)
        UCHIJAA GalacticraftAmunRa{0.7.1} [Amun-Ra] (AmunRa-GC-0.7.1.jar)
        UCHIJAA angermod{0.8.2} [AngerMod. Makes your Mobs angry!] (AngerMod-0.8.2.jar)
        UCHIJAA AppleCore{3.3.4} [AppleCore] (AppleCore-3.3.4.jar)
        UCHIJAA cookingforblockheads{1.3.7-GTNH} [Cooking For Blockheads] (CookingForBlockheads-1.3.7-GTNH.jar)
        UCHIJAA appliedcooking{1.2-gtnh} [AppliedCooking] (appliedcooking-1.2-gtnh.jar)
        UCHIJAA archaicfix{0.7.4} [ArchaicFix] (archaicfix-0.7.4.jar)
        UCHIJAA ArchitectureCraft{1.10.2} [ArchitectureCraft] (ArchitectureCraft-1.10.2.jar)
        UCHIJAA asielib{0.6.0} [AsieLib] (AsieLib-0.6.0.jar)
        UCHIJAA Automagy{0.28.2} [Automagy] (Automagy-1.7.10-0.28.2.jar)
        UCHIJAA Botania{1.12.1-GTNH} [Botania] (Botania-1.12.1-GTNH.jar)
        UCHIJAA Avaritia{1.57} [Avaritia] (Avaritia-1.57.jar)
        UCHIJAA wanionlib{1.9.0} [WanionLib] (WanionLib-1.9.0.jar)
        UCHIJAA avaritiaddons{1.8.4-GTNH} [Avaritiaddons] (Avaritiaddons-1.8.4-GTNH.jar)
        UCHIJAA Backpack{2.4.3-GTNH} [Backpack Editted for ModdedNetwork] (backpack-2.4.3-GTNH.jar)
        UCHIJAA Baubles|Expanded{2.0.3} [Baubles Expanded] (BaublesExpanded-2.0.3.jar)
        UCHIJAA betterbuilderswands{0.12.0-GTNH} [BetterBuildersWands] (BetterBuildersWands-0.12.0-GTNH.jar)
        UCHIJAA forgelin{1.10.0-GTNH} [Forgelin] (forgelin-1.10.0-GTNH.jar)
        UCHIJAA betterp2p{1.3.0} [BetterP2P] (betterp2p-1.3.0.jar)
        UCHIJAA betterquesting{3.6.22-GTNH} [BetterQuesting] (BetterQuesting-3.6.22-GTNH.jar)
        UCHIJAA cb4bq{3.6.22-GTNH} [Command Blocks for Better Questing] (BetterQuesting-3.6.22-GTNH.jar)
        UCHIJAA bq_standard{3.6.22-GTNH} [Standard Expansion] (BetterQuesting-3.6.22-GTNH.jar)
        UCHIJAA questbook{3.6.22-GTNH} [BetterQuesting] (BetterQuesting-3.6.22-GTNH.jar)
        UCHIJAA BiblioCraft{1.11.7} [BiblioCraft] (BiblioCraft[v1.11.7][MC1.7.10].jar)
        UCHIJAA BiblioWoodsBoP{1.9} [BiblioWoods Biomes O'Plenty Edition] (BiblioWoods[BiomesOPlenty][v1.9].jar)
        UCHIJAA BiblioWoodsForestry{1.7} [BiblioWoods Forestry Edition] (BiblioWoods[Forestry][v1.7].jar)
        UCHIJAA BiblioWoodsNatura{1.5} [BiblioWoods Natura Edition] (BiblioWoods[Natura][v1.5].jar)
        UCHIJAA BinnieCore{2.4.4} [Binnie Core] (binnie-mods-2.4.4.jar)
        UCHIJAA Botany{2.4.4} [Botany] (binnie-mods-2.4.4.jar)
        UCHIJAA Genetics{2.4.4} [Genetics] (binnie-mods-2.4.4.jar)
        UCHIJAA ExtraTrees{2.4.4} [Extra Trees] (binnie-mods-2.4.4.jar)
        UCHIJAA ExtraBees{2.4.4} [Extra Bees] (binnie-mods-2.4.4.jar)
        UCHIJAA blocklimiter{0.6.0} [BlockLimiter. Restrict your Blocks] (BlockLimiter-0.6.0.jar)
        UCHIJAA structurelib{1.4.1-pre} [StructureLib] (structurelib-1.4.1-pre.jar)
        UCHIJAA blockrenderer6343{1.2.17-pre} [BlockRenderer6343] (blockrenderer6343-1.2.17-pre.jar)
        UCHIJAA ThaumicTinkerer{2.10.2} [Thaumic Tinkerer] (ThaumicTinkerer-2.10.2.jar)
        UCHIJAA ForbiddenMagic{0.7.0-GTNH} [Forbidden Magic] (Forbidden.Magic-0.7.0-GTNH.jar)
        UCHIJAA BloodArsenal{1.3.4} [Blood Arsenal] (BloodArsenal-1.3.4.jar)
        UCHIJAA BuildCraft|Transport{7.1.39} [BC Transport] (buildcraft-7.1.39.jar)
        UCHIJAA BuildCraft|Silicon{7.1.39} [BC Silicon] (buildcraft-7.1.39.jar)
        UCHIJAA MagicBees{2.8.5-GTNH} [Magic Bees] (magicbees-2.8.5-GTNH.jar)
        UCHIJAA gendustry{1.8.3-GTNH} [Gendustry] (gendustry-1.8.3-GTNH.jar)
        UCHIJAA ggfab{5.09.51.12} [GigaGramFab] (gregtech-5.09.51.12.jar)
        UCHIJAA harvestcraft{1.2.3-GTNH} [Pam's HarvestCraft] (harvestcraft-1.2.3-GTNH.jar)
        UCHIJAA hodgepodge{2.6.4} [Hodgepodge] (hodgepodge-2.6.4.jar)
        UCHIJAA modularui{1.2.17} [ModularUI] (modularui-1.2.17.jar)
        UCHIJAA modularui2{2.1.16-1.7.10} [Modular UI 2] (modularui2-2.1.16-1.7.10.jar)
        UCHIJAA Translocator{1.2.1} [Translocator] (Translocator-1.2.1.jar)
        UCHIJAA gregtech{MC1710} [GregTech] (gregtech-5.09.51.12.jar)
        UCHIJAA witchery{0.24.1} [Witchery] (witchery-1.7.10-0.24.1.jar)
        UCHIJAA botanichorizons{1.1.7-GTNH} [botanichorizons] (BotanicHorizons-1.1.7-GTNH.jar)
        UCHIJAA BrandonsCore{1.2.0-GTNH} [Brandon's Core] (BrandonsCore-1.2.0-GTNH.jar)
        UCHIJAA bugtorch{1.2.14-GTNH} [BugTorch] (bugtorch-1.2.14-GTNH.jar)
        UCHIJAA BuildCraft|Robotics{7.1.39} [BC Robotics] (buildcraft-7.1.39.jar)
        UCHIJAA BuildCraft|Compat{7.1.17} [BuildCraft Compat] (buildcraft-compat-7.1.17.jar)
        UCHIJAA OilTweak{1.1.1} [BuildCraft Oil Tweak] (BuildCraftOilTweak-1.1.1.jar)
        UCHIJAA CarpentersBlocks{3.6.3-GTNH} [Carpenter's Blocks] (CarpentersBlocks-3.6.3-GTNH.jar)
        UCHIJAA catwalks{2.0.4} [Catwalks Mod] (catwalks-2.3.2-GTNH.jar)
        UCHIJAA Ztones{1.7.10} [Ztones] (Ztones-1.7.10-2.2.2.jar)
        UCHIJAA chiseltones{1.2.0-GTNH} [Chisel Tones] (chiseltones-1.2.0-GTNH.jar)
        UCHIJAA compactkineticgenerators{1.7.10-1.0} [Compact Kinetic Wind and Water Generators] (CompactKineticGenerators-1.7.10-1.0.jar)
        UCHIJAA EnderStorage{1.6.4} [EnderStorage] (EnderStorage-1.6.4.jar)
        UCHIJAA MrTJPCoreMod{1.2.1} [MrTJPCore] (MrTJPCore-1.2.1.jar)
        UCHIJAA ProjRed|Core{4.10.5-GTNH} [ProjectRed Core] (ProjRed-4.10.5-GTNH.jar)
        UCHIJAA ProjRed|Transmission{4.10.5-GTNH} [ProjectRed Transmission] (ProjRed-4.10.5-GTNH.jar)
        UCHIJAA OpenComputers{1.11.1-GTNH} [OpenComputers] (OpenComputers-1.11.1-GTNH.jar)
        UCHIJAA computronics{1.8.5-GTNH} [Computronics] (Computronics-1.8.5-GTNH.jar)
        UCHIJAA coretweaks{0.3.3.2} [CoreTweaks] (CoreTweaks-1.7.10-0.3.3.2+nomixin.jar)
        UCHIJAA cosmeticarmorreworked{1.7.10-v7} [CosmeticArmorReworked] (CosmeticArmorReworked-1.7.10-v7.jar)
        UCHIJAA MineTweaker3{3.4.0} [MineTweaker 3] (CraftTweaker-3.4.0.jar)
        UCHIJAA creativecore{1.4.0-GTNH} [CreativeCore] (creativecore-1.4.0-GTNH.jar)
        UCHIJAA DummyCore{1.19.0} [DummyCore] (DummyCore-1.19.0.jar)
        UCHIJAA thaumicbases{1.8.0} [Thaumic Bases] (Thaumic-Based-1.8.0.jar)
        UCHIJAA croploadcore{0.2.0} [CropLoadCore] (CropLoadCore-0.2.0.jar)
        UCHIJAA berriespp{1.8.0} [Crops++] (CropsPP-1.8.0.jar)
        UCHIJAA DraconicEvolution{1.3.14-GTNH} [Draconic Evolution] (Draconic-Evolution-1.3.14-GTNH.jar)
        UCHIJAA dualhotbar{1.6} [Dual Hotbar] (dualhotbar-1.7.10-1.6.jar)
        UCHIJAA EMT{1.6.0} [Electro-Magic Tools] (EMT-1.6.0.jar)
        UCHIJAA EnderZoo{1.2.0} [Ender Zoo] (EnderZoo-1.2.0.jar)
        UCHIJAA enhancedlootbags{1.2.0} [Enhanced LootBags] (EnhancedLootBags-1.2.0.jar)
        UCHIJAA universalsingularities{8.8.0} [UniversalSingularities] (Universal-Singularities-8.8.0.jar)
        UCHIJAA eternalsingularity{1.2.1} [Eternal Singularity] (eternalsingularity-1.2.1.jar)
        UCHIJAA falsepatternlib{1.4.6} [FalsePatternLib] (falsepatternlib-mc1.7.10-1.4.6.jar)
        UCHIJAA findit{1.3.10} [FindIt] (findit-1.3.10.jar)
        UCHIJAA FloodLights{1.4.2} [Flood Lights] (FloodLights-1.4.2.jar)
        UCHIJAA McMultipart{1.5.0} [Minecraft Multipart Plugin] (ForgeMultipart-1.5.0.jar)
        UCHIJAA ForgeMicroblock{1.5.0} [Forge Microblocks] (ForgeMultipart-1.5.0.jar)
        UCHIJAA NBTEdit{1.7.10} [In-game NBTEdit] (ForgeNBTEdit-universal-1.0.0.test.jar)
        UCHIJAA ForgeRelocation{0.2.0} [ForgeRelocation] (ForgeRelocation-0.2.0.jar)
        UCHIJAA MCFrames{0.2.0} [MCFrames] (ForgeRelocation-0.2.0.jar)
        UCHIJAA RelocationFMP{0.2.0} [RelocationFMP] (ForgeRelocationFMP-0.2.0.jar)
        UCHIJAA gadomancy{1.3.6} [Gadomancy] (gadomancy-1.3.6.jar)
        UCHIJAA gtneioreplugin{5.09.51.12} [GT NEI Ore Plugin GT:NH Mod] (gregtech-5.09.51.12.jar)
        UCHIJAA tectech{5.09.51.12} [TecTech - Tec Technology!] (gregtech-5.09.51.12.jar)
        UCHIJAA ProjRed|Illumination{4.10.5-GTNH} [ProjectRed Illumination] (ProjRed-4.10.5-GTNH.jar)
        UCHIJAA RandomThings{2.5.3} [RandomThings] (RandomThings-2.5.3.jar)
        UCHIJAA bartworks{5.09.51.12} [BartWorks] (gregtech-5.09.51.12.jar)
        UCHIJAA GoodGenerator{5.09.51.12} [Good Generator] (gregtech-5.09.51.12.jar)
        UCHIJAA gtnhlanth{5.09.51.12} [GTNH: Lanthanides] (gregtech-5.09.51.12.jar)
        UCHIJAA bartworkscrossmodtgregworkscontainer{5.09.51.12} [BartWorks Mod Additions - TGregworks Container] (gregtech-5.09.51.12.jar)
        UCHIJAA IC2NuclearControl{2.6.7} [Nuclear Control 2] (IC2NuclearControl-2.6.7.jar)
        UCHIJAA OpenMods{0.10.9} [OpenMods] (OpenModsLibs-0.10.9.jar)
        UCHIJAA OpenBlocks{1.10.2-GTNH} [OpenBlocks] (OpenBlocks-1.10.2-GTNH.jar)
        UCHIJAA StevesCarts{2.2.0} [Steve's Carts 2] (StevesCarts-2.2.0.jar)
        UCHIJAA TGregworks{1.7.10-GTNH-1.0.27} [Tinkers' Gregworks] (TGregworks-1.7.10-GTNH-1.0.27.jar)
        UCHIJAA miscutils{5.09.51.12} [GT++] (gregtech-5.09.51.12.jar)
        UCHIJAA GalaxySpace{1.1.100-GTNH} [GalaxySpace] (GalaxySpace-1.1.100-GTNH.jar)
        UCHIJAA GraviSuite{1.7.10-2.0.3} [Graviation Suite] (GraviSuite-1.7.10-2.0.3.jar)
        UCHIJAA gravisuiteneo{1.2.2} [Gravitation Suite Neo] (gravisuiteneo-1.2.2.jar)
        UCHIJAA kekztech{5.09.51.12} [KekzTech] (gregtech-5.09.51.12.jar)
        UCHIJAA detravscannermod{5.09.51.12} [GT Scanner Mod] (gregtech-5.09.51.12.jar)
        UCHIJAA InfernalMobs{1.10.0-GTNH} [Infernal Mobs] (InfernalMobs-1.10.0-GTNH.jar)
        UCHIJAA mobsinfo{0.5.0-GTNH} [MobsInfo] (mobsinfo-0.5.0-GTNH.jar)
        UCHIJAA kubatech{5.09.51.12} [KubaTech] (gregtech-5.09.51.12.jar)
        UCHIJAA galacticgreg{5.09.51.12} [Galactic Greg] (gregtech-5.09.51.12.jar)
        UCHIJAA bartworkscrossmod{5.09.51.12} [BartWorks Mod Additions] (gregtech-5.09.51.12.jar)
        UCHIJAA ToxicEverglades{5.09.51.12} [GregTech ToxicEverglades] (gregtech-5.09.51.12.jar)
        UCHIJAA TaintedMagic{r7.6} [Tainted Magic] (Tainted-Magic-7.6.13-GTNH.jar)
        UCHIJAA ThaumicExploration{1.3.7-GTNH} [Thaumic Exploration] (Thaumic-Exploration-1.3.7-GTNH.jar)
        UCHIJAA gtnhtcwands{1.4.1} [GTNH-TC-Wands] (GTNH-TC-Wands-1.4.1.jar)
        UCHIJAA openmodularturrets{2.4.0} [Open Modular Turrets] (OpenModularTurrets-2.4.0.jar)
        UCHIJAA gtnhintergalactic{1.5.2} [GTNH-Intergalactic] (gtnhintergalactic-1.5.2.jar)
        UCHIJAA GTNHModify_CutCorners{v1.2.0-cutcorners-master.1+74e2ae752e-dirty} [GTNHModify] (GTNHModify_CutCorners-v1.2.1.jar)
        UCHIJAA GTTweaker{2.2.4} [GTTweaker] (GTTweaker-2.2.4.jar)
        UCHIJAA HardcoreDarkness{1.7} [Hardcore Darkness] (HardcoreDarkness-MC1.7.10-1.7.jar)
        UCHIJAA HelpFixer{1.3.0} [HelpFixer] (HelpFixer-1.3.0.jar)
        UCHIJAA holoinventory{2.4.13-GTNH} [Holo Inventory] (holoinventory-2.4.13-GTNH.jar)
        UCHIJAA hydroenergy{1.3.6} [HydroEnergy] (hydroenergy-1.3.6.jar)
        UCHIJAA ifu{1.11.1} [I Will Find You] (ifu-1.11.1.jar)
        UCHIJAA LunatriusCore{1.2.1-GTNH} [LunatriusCore] (LunatriusCore-1.2.1-GTNH.jar)
        UCHIJAA InGameInfoXML{2.8.19} [InGame Info XML] (InGameInfoXML-2.8.19.jar)
        UCHIJAA ironchestminecarts{1.1.0} [Iron Chest Minecarts] (IronChestMinecarts-1.1.0.jar)
        UCHIJAA irontank{1.3.0} [Iron Tanks] (irontanks-1.3.0.jar)
        UCHIJAA JABBA{1.4.6} [JABBA] (JABBA-1.4.6.jar)
        UCHIJAA JourneyMapServer{1.0.5_MC1.7.10} [JourneyMapServer] (JourneyMapServer1.0.5_MC1.7.10.jar)
        UCHIJAA laggoggles{4.16.1} [LagGoggles] (laggoggles-mc1.7.10-4.16.1.jar)
        UCHIJAA littletiles{1.3.1-GTNH} [LittleTiles] (littletiles-1.3.1-GTNH.jar)
        UCHIJAA LogisticsPipes{1.2.34-GTNH} [Logistics Pipes] (logisticspipes-1.2.34-GTNH.jar)
        UCHIJAA lootgames{2.1.4} [LootGames] (lootgames-2.1.4.jar)
        UCHIJAA lwjgl3ify{2.1.5} [lwjgl3ify] (lwjgl3ify-2.1.5.jar)
        UCHIJAA malisiscore{1.17.4-GTNH} [Malisis Core] (malisisdoors-1.17.4-GTNH.jar)
        UCHIJAA malisisdoors{1.17.4-GTNH} [Malisis' Doors] (malisisdoors-1.17.4-GTNH.jar)
        UCHIJAA mkb{2.1.0} [ModernKeyBinding] (ModernKeyBinding-Forge-1.7.10-2.1.0-mixinless.jar)
        UCHIJAA modtweaker2{0.11.0} [Mod Tweaker 2] (ModTweaker2-0.11.0.jar)
        UCHIJAA Morpheus{1.7.10-1.6.21} [Morpheus] (Morpheus-1.7.10-1.6.21.jar)
        UCHIJAA mxrandom{0.5.1} [MX-Random] (mxrandom-0.5.1.jar)
        UCHIJAA naturescompass{1.4.4-GTNH} [Nature's Compass] (naturescompass-1.4.4-GTNH.jar)
        UCHIJAA navigator{1.0.15} [Navigator] (navigator-1.0.15.jar)
        UCHIJAA NEIAddons{1.16.0} [NEI Addons] (NEIAddons-1.16.0.jar)
        UCHIJAA NEIAddons|Forestry{1.16.0} [NEI Addons: Forestry] (NEIAddons-1.16.0.jar)
        UCHIJAA NEIAddons|Developer{1.16.0} [NEI Addons: Developer Tools] (NEIAddons-1.16.0.jar)
        UCHIJAA NEIAddons|AppEng{1.16.0} [NEI Addons: Applied Energistics 2] (NEIAddons-1.16.0.jar)
        UCHIJAA NEIAddons|Botany{1.16.0} [NEI Addons: Botany] (NEIAddons-1.16.0.jar)
        UCHIJAA NEIAddons|ExNihilo{1.16.0} [NEI Addons: Ex Nihilo] (NEIAddons-1.16.0.jar)
        UCHIJAA NEIAddons|CraftingTables{1.16.0} [NEI Addons: Crafting Tables] (NEIAddons-1.16.0.jar)
        UCHIJAA neicustomdiagram{1.7.1} [NEI Custom Diagram] (NEICustomDiagram-1.7.1.jar)
        UCHIJAA neiintegration{1.4.0} [NEIIntegration] (NEIIntegration-1.4.0.jar)
        UCHIJAA netherportalfix{1.3.0} [Nether Portal Fix] (netherportalfix-1.3.0.jar)
        UCHIJAA NodalMechanics{1.2.1-GTNH} [NodalMechanics] (NodalMechanics-1.2.1-GTNH.jar)
        UCHIJAA neenergistics{1.6.4} [NotEnoughEnergistics] (NotEnoughEnergistics-1.6.4.jar)
        UCHIJAA neid{2.1.6} [NotEnoughIDs] (notenoughIDs-2.1.6.jar)
        UCHIJAA nutrition{0.0.13} [Nutrition] (nutrition-0.0.13.jar)
        UCHIJAA OmniOcularUnofficial{GRADLETOKEN_VERSION} [Omni Ocular Unofficial] (OmniOcularUnofficial-1.5.jar)
        UCHIJAA openglasses{1.5.0-GTNH} [OC Glasses] (OpenGlasses-1.5.0-GTNH.jar)
        UCHIJAA openprinter{0.3.0-GTNH} [OpenPrinter] (openprinter-0.3.0-GTNH.jar)
        UCHIJAA opensecurity{1.1.1-GTNH} [OpenSecurity] (opensecurity-1.1.1-GTNH.jar)
        UCHIJAA Opis{1.4.5-mapless} [Opis] (Opis-1.4.5-mapless.jar)
        UCHIJAA harvestthenether{1.7.10} [Pam's Harvest the Nether] (Pam's Harvest the Nether 1.7.10a.jar)
        UCHIJAA personalspace{1.0.32} [PersonalSpace] (personalspace-1.0.32.jar)
        UCHIJAA postea{1.0.13} [Postea] (postea-1.0.13.jar)
        UCHIJAA programmablehatches{0.1.1p2} [ProgrammableHatches] (programmablehatches-0.1.1p2.jar)
        UCHIJAA ProjectBlue{1.2.0-GTNH} [Project Blue] (ProjectBlue-1.2.0-GTNH.jar)
        UCHIJAA ProjRed|Integration{4.10.5-GTNH} [ProjectRed Integration] (ProjRed-4.10.5-GTNH.jar)
        UCHIJAA ProjRed|Expansion{4.10.5-GTNH} [ProjectRed Expansion] (ProjRed-4.10.5-GTNH.jar)
        UCHIJAA ProjRed|Transportation{4.10.5-GTNH} [ProjectRed Transportation] (ProjRed-4.10.5-GTNH.jar)
        UCHIJAA ProjRed|Exploration{4.10.5-GTNH} [ProjectRed Exploration] (ProjRed-4.10.5-GTNH.jar)
        UCHIJAA ProjRed|Compatibility{4.10.5-GTNH} [ProjectRed Compatibility] (ProjRed-4.10.5-GTNH.jar)
        UCHIJAA ProjRed|Fabrication{4.10.5-GTNH} [ProjectRed Fabrication] (ProjRed-4.10.5-GTNH.jar)
        UCHIJAA qz_miner{v1.9.0-beta.3} [qz_miner-鐖嗙牬杩為攣] (qz_miner-v1.9.0-beta.3.jar)
        UCHIJAA RIO{2.6.4} [RemoteIO] (RemoteIO-2.6.4.jar)
        UCHIJAA Roguelike{1.6.0-GTNH} [Roguelike Dungeons] (roguelike-1.6.0-GTNH.jar)
        UCHIJAA RWG{alpha-1.5.0} [Realistic World Gen Alpha] (RWG-alpha-1.5.0.jar)
        UCHIJAA serverutilities{2.1.4} [ServerUtilities] (ServerUtilities-2.1.4.jar)
        UCHIJAA SGCraft{1.4.4-GTNH} [SG Craft] (SGCraft-1.4.4-GTNH.jar)
        UCHIJAA sharewhereiam{2.1.1} [Share Where I am] (sharewhereiam-2.1.1.jar)
        UCHIJAA skinport{1.7.10-v10d} [SkinPort] (SkinPort-1.7.10-v10d.jar)
        UCHIJAA sleepingbag{0.2.1} [Sleeping Bag] (sleepingbag-0.2.1.jar)
        UCHIJAA SpecialMobs{3.6.2} [Special Mobs] (SpecialMobs-3.6.2.jar)
        UCHIJAA SpiceOfLife{2.1.12-carrot} [The Spice of Life - Carrot Edition] (SpiceOfLife-2.1.12-carrot.jar)
        UCHIJAA StevesFactoryManager{1.3.2-GTNH} [Steve's Factory Manager] (StevesFactoryManager-1.3.2-GTNH.jar)
        UCHIJAA StevesAddons{0.13.0} [Steve's Addons] (StevesAddons-0.13.0.jar)
        UCHIJAA StorageDrawers{2.0.5-GTNH} [Storage Drawers] (StorageDrawers-2.0.5-GTNH.jar)
        UCHIJAA StorageDrawersErebus{2.0.5-GTNH} [Storage Drawers: Erebus Pack Pack] (StorageDrawers-2.0.5-GTNH.jar)
        UCHIJAA StorageDrawersForestry{2.0.5-GTNH} [Storage Drawers: Forestry Pack] (StorageDrawers-2.0.5-GTNH.jar)
        UCHIJAA StorageDrawersBop{2.0.5-GTNH} [Storage Drawers: Biomes O' Plenty Pack] (StorageDrawers-2.0.5-GTNH.jar)
        UCHIJAA StorageDrawersMisc{2.0.5-GTNH} [Storage Drawers: Misc Pack] (StorageDrawers-2.0.5-GTNH.jar)
        UCHIJAA StorageDrawersNatura{2.0.5-GTNH} [Storage Drawers: Natura Pack] (StorageDrawers-2.0.5-GTNH.jar)
        UCHIJAA structurecompat{0.6.5} [StructureCompat] (structurecompat-0.6.5.jar)
        UCHIJAA SuperTic{1.4.0} [SuperTic] (SuperTic-1.4.0.jar)
        UCHIJAA tcinventoryscan{1.2.0-GTNH} [TC Inventory Scanning] (tcinventoryscan-1.2.0-GTNH.jar)
        UCHIJAA thaumcraftneiplugin{1.7.10-1.7a} [Thaumcraft NEI Plugin] (thaumcraftneiplugin-1.7.10-1.7a.jar)
        UCHIJAA tcneiadditions{1.4.2} [Thaumcraft NEI Additions] (tcneiadditions-1.4.2.jar)
        UCHIJAA tc4tweak{1.5.24} [TC4 Tweak] (Thaumcraft4Tweaks-1.5.24.jar)
        UCHIJAA ThaumcraftMobAspects{1.1.2-GTNH} [Thaumcraft Mob Aspects] (ThaumcraftMobAspects-1.1.2-GTNH.jar)
        UCHIJAA ThaumcraftResearchTweaks{1.2.0} [Thaumcraft Research Tweaks] (ThaumcraftResearchTweaks-1.2.0.jar)
        UCHIJAA ThaumicMachina{0.2.1} [Thaumic Machina] (Thaumic Machina-1.7.10-0.2.1.jar)
        UCHIJAA thaumicboots{1.3.10} [Thaumic Boots] (thaumicboots-1.3.10.jar)
        UCHIJAA ThaumicHorizons{1.6.5} [ThaumicHorizons] (ThaumicHorizons-1.6.5.jar)
        UCHIJAA thaumicinsurgence{0.3.1} [Thaumic Insurgence] (thaumicinsurgence-0.3.1.jar)
        UCHIJAA tinkersdefense{1.2.2} [Tinkers' Defense] (tinkersdefense-1.2.2.jar)
        UCHIJAA TMechworks{0.3.1} [Tinkers Mechworks] (TMechworks-0.3.1.jar)
        UCHIJAA TML{4.2.0-GTNH} [TooMuchLoot] (TooMuchLoot-4.2.0-GTNH.jar)
        UCHIJAA travellersgearneo{1.0} [TravellersGearNeo] (travellersgearneo-1.2.4.jar)
        UCHIJAA TravellersGear{1.16.6} [Traveller's Gear] (TravellersGear-1.7.10-1.16.6.jar)
        UCHIJEE TwistSpaceTechnology{0.6.0} [Twist Space Technology] (TwistSpaceTechnology-0.6.0.jar)
        UCHIJAA visualprospecting{1.3.28} [VisualProspecting] (visualprospecting-1.3.28.jar)
        UCHIJAA WailaHarvestability{1.2.1-GTNH} [Waila Harvestability] (WailaHarvestability-1.2.1-GTNH.jar)
        UCHIJAA wailaplugins{0.6.0} [WAILA Plugins] (WAILAPlugins-0.6.0.jar)
        UCHIJAA WarpTheory{1.4.3-GTNH} [WarpTheory] (WarpTheory-1.4.3-GTNH.jar)
        UCHIJAA wawla{1.3.0-GTNH} [What Are We Looking At] (Wawla-1.3.0-GTNH.jar)
        UCHIJAA WitcheryExtras{1.2.3} [Witchery++] (WitcheryExtras-1.2.3.jar)
        UCHIJAA WitchingGadgets{1.5.17-GTNH} [Witching Gadgets] (WitchingGadgets-1.5.17-GTNH.jar)
        UCHIJAA worldedit{%VERSION%} [WorldEdit] (worldedit-v0.0.7.jar)
        UCHIJAA WR-CBE|Core{1.6.1} [WR-CBE Core] (WR-CBE-1.6.1.jar)
        UCHIJAA WR-CBE|Logic{1.6.1} [WR-CBE Logic] (WR-CBE-1.6.1.jar)
        UCHIJAA WR-CBE|Addons{1.6.1} [WR-CBE Addons] (WR-CBE-1.6.1.jar)
        UCHIJAA HungerOverhaul{1.7.10-1.0.0.jenkins104} [Hunger Overhaul] (HungerOverhaul-1.7.10-1.0.0.jenkins104.jar)
        UCHIJAA IguanaTweaksTConstruct{2.5.0} [Iguana Tinker Tweaks] (IguanaTweaksTConstruct-2.5.0.jar)
[12:51:57] [Server thread/ERROR] [FML]: The following problems were captured during this phase
[12:51:57] [Server thread/ERROR] [FML]: Caught exception from TwistSpaceTechnology
java.lang.NoClassDefFoundError: galaxyspace/core/register/GSMaterials
        at Launch//com.Nxer.TwistSpaceTechnology.recipe.machineRecipe.MixerRecipePool.loadRecipes(MixerRecipePool.java:22) ~[MixerRecipePool.class:?]
        at Launch//com.Nxer.TwistSpaceTechnology.loader.RecipeLoader.loadRecipes(RecipeLoader.java:76) ~[RecipeLoader.class:?]
        at Launch//com.Nxer.TwistSpaceTechnology.TwistSpaceTechnology.completeInit(TwistSpaceTechnology.java:134) ~[TwistSpaceTechnology.class:?]
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at Launch//cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532) ~[FMLModContainer.class:?]
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at Launch//com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[guava-17.0.jar:?]
        at Launch//cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212) ~[LoadController.class:?]
        at Launch//cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190) ~[LoadController.class:?]
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at Launch//com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[guava-17.0.jar:?]
        at Launch//cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119) [LoadController.class:?]
        at Launch//cpw.mods.fml.common.Loader.initializeMods(Loader.java:745) [Loader.class:?]
        at Launch//cpw.mods.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:97) ~[FMLServerHandler.class:?]
        at Launch//cpw.mods.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:319) [FMLCommonHandler.class:?]
        at Launch//net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:210) [lt.class:?]
        at Launch//net.minecraft.server.MinecraftServer.run(MinecraftServer.java:387) [MinecraftServer.class:?]
        at Launch//net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685) [li.class:?]
Caused by: java.lang.ClassNotFoundException: Class bytes are null for galaxyspace.core.register.GSMaterials (galaxyspace.core.register.GSMaterials, galaxyspace.core.register.GSMaterials)
        at System//net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:325) ~[LaunchClassLoader.class:?]
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
        ... 29 more
[12:51:57] [Server thread/ERROR] [FML]: A fatal exception occurred during the server about to start event
cpw.mods.fml.common.LoaderException: java.lang.NoClassDefFoundError: galaxyspace/core/register/GSMaterials

        at Launch//cpw.mods.fml.common.LoadController.transition(LoadController.java:163) ~[LoadController.class:?]
        at Launch//cpw.mods.fml.common.Loader.serverAboutToStart(Loader.java:860) [Loader.class:?]
        at Launch//cpw.mods.fml.common.FMLCommonHandler.handleServerAboutToStart(FMLCommonHandler.java:273) [FMLCommonHandler.class:?]
        at Launch//net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:256) [lt.class:?]
        at Launch//net.minecraft.server.MinecraftServer.run(MinecraftServer.java:387) [MinecraftServer.class:?]
        at Launch//net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685) [li.class:?]
Caused by: java.lang.NoClassDefFoundError: galaxyspace/core/register/GSMaterials
        at Launch//com.Nxer.TwistSpaceTechnology.recipe.machineRecipe.MixerRecipePool.loadRecipes(MixerRecipePool.java:22) ~[MixerRecipePool.class:?]
        at Launch//com.Nxer.TwistSpaceTechnology.loader.RecipeLoader.loadRecipes(RecipeLoader.java:76) ~[RecipeLoader.class:?]
        at Launch//com.Nxer.TwistSpaceTechnology.TwistSpaceTechnology.completeInit(TwistSpaceTechnology.java:134) ~[TwistSpaceTechnology.class:?]
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at Launch//cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532) ~[FMLModContainer.class:?]
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at Launch//com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[guava-17.0.jar:?]
        at Launch//cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212) ~[LoadController.class:?]
        at Launch//cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190) ~[LoadController.class:?]
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at Launch//com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[guava-17.0.jar:?]
        at Launch//cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119) ~[LoadController.class:?]
        at Launch//cpw.mods.fml.common.Loader.initializeMods(Loader.java:745) ~[Loader.class:?]
        at Launch//cpw.mods.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:97) ~[FMLServerHandler.class:?]
        at Launch//cpw.mods.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:319) ~[FMLCommonHandler.class:?]
        at Launch//net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:210) ~[lt.class:?]
        ... 2 more
Caused by: java.lang.ClassNotFoundException: Class bytes are null for galaxyspace.core.register.GSMaterials (galaxyspace.core.register.GSMaterials, galaxyspace.core.register.GSMaterials)
        at System//net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:325) ~[LaunchClassLoader.class:?]
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
        at Launch//com.Nxer.TwistSpaceTechnology.recipe.machineRecipe.MixerRecipePool.loadRecipes(MixerRecipePool.java:22) ~[MixerRecipePool.class:?]
        at Launch//com.Nxer.TwistSpaceTechnology.loader.RecipeLoader.loadRecipes(RecipeLoader.java:76) ~[RecipeLoader.class:?]
        at Launch//com.Nxer.TwistSpaceTechnology.TwistSpaceTechnology.completeInit(TwistSpaceTechnology.java:134) ~[TwistSpaceTechnology.class:?]
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at Launch//cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532) ~[FMLModContainer.class:?]
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at Launch//com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[guava-17.0.jar:?]
        at Launch//cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212) ~[LoadController.class:?]
        at Launch//cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190) ~[LoadController.class:?]
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
        at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
        at Launch//com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[guava-17.0.jar:?]
        at Launch//com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[guava-17.0.jar:?]
        at Launch//cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119) ~[LoadController.class:?]
        at Launch//cpw.mods.fml.common.Loader.initializeMods(Loader.java:745) ~[Loader.class:?]
        at Launch//cpw.mods.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:97) ~[FMLServerHandler.class:?]
        at Launch//cpw.mods.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:319) ~[FMLCommonHandler.class:?]
        at Launch//net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:210) ~[lt.class:?]
        ... 2 more
```

--- a/src/main/java/com/Nxer/TwistSpaceTechnology/recipe/machineRecipe/MixerRecipePool.java
+++ b/src/main/java/com/Nxer/TwistSpaceTechnology/recipe/machineRecipe/MixerRecipePool.java
@@ -2,13 +2,14 @@ package com.Nxer.TwistSpaceTechnology.recipe.machineRecipe;
 
 import static com.Nxer.TwistSpaceTechnology.util.enums.TierEU.RECIPE_EV;
 import static com.Nxer.TwistSpaceTechnology.util.enums.TierEU.RECIPE_LuV;
-import static galaxyspace.core.register.GSMaterials.tantalumCarbideHafniumCarbideMixture;
 
 import com.Nxer.TwistSpaceTechnology.recipe.IRecipePool;
 
 import gregtech.api.enums.GTValues;
 import gregtech.api.enums.Materials;
+import gregtech.api.enums.Mods;
 import gregtech.api.enums.OrePrefixes;
+import gregtech.api.util.GTModHandler;
 import gregtech.api.util.GTUtility;
 import gtPlusPlus.api.recipe.GTPPRecipeMaps;
 import gtnhlanth.common.register.WerkstoffMaterialPool;
@@ -24,7 +25,7 @@ public class MixerRecipePool implements IRecipePool {
                 WerkstoffMaterialPool.Hafnium.get(OrePrefixes.dust, 1),
                 Materials.Tantalum.getDust(4),
                 Materials.Carbon.getDust(5))
-            .itemOutputs(tantalumCarbideHafniumCarbideMixture.get(OrePrefixes.dust, 10))
+            .itemOutputs(GTModHandler.getModItem(Mods.BartWorks.ID, "gt.bwMetaGenerateddust", 10, 11502))
             .noOptimize()
             .eut(RECIPE_EV)
             .duration(20 * 10)

